config/webpack.config.js

```diff
@@ -293,6 +293,7 @@ module.exports = async (env = {}) => {
             options: {
               // Speed up compilation.
               cacheDirectory: '.babelcache',
+              cacheCompression: false,
```

Comment:
What's the main reason to disabled the `cacheCompression`? would this make the cache bigger but faster?

Reply:
Yes, this is just to make it faster.